lib/app/widgets/app_bar.dart

```diff
@@ -15,11 +17,14 @@ class _MyAppBarState extends State<MyAppBar> {
   Future<void> _showMenu() async {
     String targetScreen = await showModalBottomSheet(
       context: context,
-      builder: (context) => Menu(),
+      builder: (context) =>
+          Menu(navigation: Provider.of<NavigationService>(context)),
```

Comment:
Shouldn't we rather just pass in the route, thereby limiting the scope of the state?